pkg/blob/controllerserver.go

```diff
@@ -163,6 +163,8 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 		if !storeAccountKey {
 			return nil, status.Errorf(codes.InvalidArgument, fmt.Sprintf("storeAccountKey must set as true in cross subscription(%s)", subsID))
 		}
+	} else {
+		subsID = d.cloud.SubscriptionID
```

Comment:
it should be ok set `subsID` as empty which mean this driver would use default subsID as current cluster, blob client supports `subsID` as empty, right?

Reply:
That's right. I will leave it empty.